.azure-pipelines/linux-conda-CI.yml

```diff
@@ -29,12 +31,17 @@ jobs:
       packageSpecs: 'python=$(python.version)'
 
   - script: |
-      conda config --set always_yes yes --set changeps1 no
-      conda install -c conda-forge protobuf
-      conda install -c conda-forge numpy
-      python -m pip install --upgrade pip
+      test '$(python.version)' == '3.7' && apt-get install protobuf-compiler libprotoc-dev
+      conda config --set always_yes yes --set changeps1 no      
+      conda install protobuf numpy
+      python -m pip install --upgrade pip      
+      test '$(python.version)' == '3.7' && git clone https://github.com/onnx/onnx.git --recursive
```

Comment:
It's pure bash script here, you can merge them into if statement.

Reply:
done